### PR TITLE
feat: kiva classic typography for /portfolio/lending-stats

### PR DESCRIPTION
**Pull request by ryan-ludwig**
http://test.kiva.org/portfolio/lending-stats
http://www.kiva.org/portfolio/lending-stats

### CODE REVIEW

File: src/pages/LendingStats/LendingStatsPage.vue [hunk 1]
```diff
@@ -3,58 +3,62 @@
...
-				/>
-				<hr>
-				<p>
-					* Please note, Kiva is continually adding and ending partnerships as we deem necessary.
-					This means, you may end up supporting a loan in a country or through a Field Partner that
-					is no longer active and therefore not included in the total number of countries or partners
-					noted on this page. For this reason, it is possible to see discrepancies between the number
-					you've supported and the number to go.
-				</p>
+		<kv-default-wrapper>
```
Comment (mcstover) [inline src/pages/LendingStats/LendingStatsPage.vue:6]: Wondering if this makes this page drastically different in layout than the other portfolio pages???

Comment (ryan-ludwig) [reply src/pages/LendingStats/LendingStatsPage.vue:6]: Yeah, it's a little different for sure. Though this whole area is not particularly consistent. What do you think?

Screen Shot 2021-11-15 at 5 46 47 PM

Screen Shot 2021-11-15 at 5 46 35 PM

Screen Shot 2021-11-15 at 5 47 18 PM

Screen Shot 2021-11-15 at 5 47 01 PM

Comment (mcstover) [reply src/pages/LendingStats/LendingStatsPage.vue:6]: Looks fine. I was mostly thinking of the width of the container switching up a new way. The extra space at the top seems a little much but we can go with it...

**Review by mcstover — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
Nice! btw, this was the first ever ui-server page, so it's fun to see it refreshed!

**Pull request merged**